.k8/helm3/ssv-liquidator/stage-values.yaml

```diff
@@ -4,8 +4,8 @@
 
 replicaCount: 
   ssv_liquidator_worker: 1
-  ssv_liquidator_worker_2: 1
-  ssv_liquidator_worker_3: 1
+  ssv_liquidator_worker_2: 0
+  ssv_liquidator_worker_3: 0
```

Comment:
Before merging to stage should:
1. Ask Stoyan to create persistent volumes configurations
2. Roll back these replicas to initial values